modules/ROOT/pages/deployment/general/general-info.adoc

```diff
@@ -0,0 +1,202 @@
+----
+ocis kill [extension name]
+----
+
+Start a extension::
+[source,bash]
+----
+ocis run [extension name]
+----
+
+==== Non Supervised Extensions
+
+You can create at any time non oCIS supervised instances of an extension with `ocis [extension name]` like `ocis proxy`. _These extensions are independent of extensions in supervised mode and have their own PID_. The Instances are managed with classical OS methods or e.g. via Kubernetes.
+
+Creating multiple instances of an extensions is used for scaling. Note that you may need configuration for and access to the instantinated extensions like with a load balancer when you scale.
+
+== Configuration of oCIS
+
+// taken from: https://owncloud.dev/ocis/config/
+
+oCIS uses a hierachical structure for its configuration, *where each element overwrites its precedent*. These are:
+
+. Environment variables
+. Extension configuration file
+. oCIS configuration file
+
+The expected loading locations are:
+
+. `$HOME/.ocis/config/`
+. `/etc/ocis/`
+. `.config/`
+
+You can define a non standard configuration file location on startup either with an:
+
+* command option (`--config-file value`) or with a
+* environment variable (`OCIS_CONFIG_FILE`)
+
+NOTE: Administrators must be aware of these sources and the order applied (the _configuration file arithmetics_). Mis-managing them can be a source of confusion leading to undesired results on the final configuration created and applied.
+
+=== Configuration File Naming
+
+The configuration files for oCIS are YAML based (a human-friendly data serialization language).
+
+The filename to define a config has following namespace:
+
+[source,plaintext]
+----
+ocis.yaml
+ or
+[extension name].yaml
+----
+
+You can list the possible extension names by typing:
+
+[source,bash]
+----
+ocis list
+----
+
+=== Starting oCIS With Environment Variables
+
+You can use environment variables to define or overwrite config parameters which will be used when starting oCIS like:
+
+[source,bash]
+----
+PROXY_HTTP_ADDR=localhost:5555 ocis server
+----
+
+or when using multiple environment variables like:
+
+[source,bash]
+----
+PROXY_HTTP_ADDR=localhost:5555 \
+PROXY_DEBUG_ADDR=localhost:6666 \
+ocis server
+----
+
+Remember the note in xref:ocis_supervised_extensions[oCIS Supervised Extensions] when killing/restarting extensions in superviced mode.
+
+=== Globally Shared Logging Values
+
+When running in supervised mode (`ocis server`), it is beneficial to have common values for logging so that the log output is correctly formatted or everything is piped to the same file without duplicating config keys and values all over the place. This is possible using the global log config key with following example:
+
+.ocis.yaml
+[source,yaml]
+----
+log:
+  level: error
+  color: true
+  pretty: true
+  file: /var/tmp/ocis_output.log
+----
+
+NOTE: In case of an extension overwriting its shared logging config that received from the main ocis.yaml file, you *MUST* specify all values.
```

Comment:
```suggestion
NOTE: In case of an extension overwriting its shared logging config received from the main ocis.yaml file, you *MUST* specify all values.
```